src/main/java/com/nlocketz/maven/plugin/JavaEntryMojo.java

```diff
@@ -45,6 +45,9 @@
 	@Parameter(defaultValue = "${project.build.directory}/entry-points.txt", property = "outputFile", required = false)
 	private String outputFilePath;
 
+	@Parameter(defaultValue = "**.*")
+	private String includeFilter;
+
```

Comment:
In the future we should consider allowing the user to define different filters (like exclude) and also multiple.